pennylane/devices/__init__.py

```diff
@@ -29,6 +29,9 @@
     default_qubit_autograd
     default_gaussian
     default_mixed
+    tf_ops
+    torch_ops
+    autograd_ops
```

Comment:
How come these lines were added here? These will likely be the reason for the failing docs CI check ([see here](https://github.com/PennyLaneAI/pennylane/runs/4451739846?check_suite_focus=true#step:4:1494).
```suggestion
```